apps/builder/app/canvas/features/webstudio-component/webstudio-component.tsx

```diff
@@ -176,6 +177,10 @@ export const WebstudioComponentDev = ({
 
   const Component = components.get(instance.component);
 
+  if (show === false) {
+    return <></>;
+  }
+
   if (Component === undefined) {
     return <></>;
   }
```

Comment:
Why not `null`? Do we need empty fragment?

Reply:
Because typescript sucks at jsx. Component returning null is not considered component.